keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -101,6 +103,9 @@ class ResourceActuator(
           }
         }
       }
+    } catch (e: ResourceCurrentlyUnresolvable) {
+      log.warn("Resource check for {} failed (hopefully temporarily) due to {}", id, e.message)
+      publisher.publishEvent(ResourceCheckPending(resource, e, clock))
```

Comment:
Can you add a friendly message here, so that the stack trace doesn't show up in the history view?

Reply:
It won't. It's only actually using the message from `e` inside the event constructor